src/components/Dashboard/StaffNotation.vue

```diff
@@ -300,6 +322,44 @@ export default class StaffNotation extends Mixins(AllowableActionsMixin, FilingM
     this.close(needReload)
   }
 
+  get showConsentAmalgamateOut (): boolean {
+    return (
+      (this.isBenBcCccUlc || this.isCoop) &&
+      !!GetFeatureFlag('supported-consent-amalgamation-out-entities').includes(this.getLegalType)
+    )
+  }
```

Comment:
Ie, don't show if FF is disabled.

Meanwhile, this action is enabled according to allowable actions.